test/copy-to-s3/mzcompose.py

```diff
@@ -138,3 +159,106 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
                     actual_parquet == expected
                 ), f"Table {table}\nActual:\n{actual_parquet[0:3]}\n...\n\nExpected:\n{expected[0:3]}\n..."
                 del actual_parquet
+
+
+def workflow_ci(c: Composition, _parser: WorkflowArgumentParser) -> None:
```

Comment:
typically I see most directories just use `default` and filter the non-ci ones out in the loop in there, but your call!

Reply:
Yeah I saw that pattern for most, but it also seems that most other directories don't co-mingle nightly and non-nightly test types. I think I'll likely refactor this in another PR since I also want to pull in the copy-to-s3 tests currently in src/test/testdrive but don't want the diff of this PR to be confusing

Reply:
sounds good!